docs/release_notes.md

```diff
@@ -14,10 +20,20 @@ For a list of all changes in this release, see the [full changelog](https://gith
 
 ### Documentation
 
+- Added information about percent bias to the "Additive Bias and Multiplicative Bias" tutorial. See [PR #639](https://github.com/nci/scores/pull/639) and [PR #656](https://github.com/nci/scores/pull/656). 
+- Updated documentation to say there are now over 60 metrics, statistical techniques and data processing tools contained in `scores`. See [PR #659](https://github.com/nci/scores/pull/659).
+- Updated instructions for installing a conda-based virtual environment in the "Contributing Guide". See [PR #654](https://github.com/nci/scores/pull/654). 
+
 ### Internal Changes
 
+- Modified automated tests to work with NumPy 2.1. Incorporated a union type of `array` and `generic` in assert statements for Dask operations. See [PR #643](https://github.com/nci/scores/pull/643).
+
 ### Contributors to this Release
 
+Durga Shrestha* ([@durgals](https://github.com/durgals)), Maree Carroll ([@mareecarroll](https://github.com/mareecarroll)), Nicholas Loveday ([@nicholasloveday](https://github.com/nicholasloveday)), Tennessee Leeuwenburg ([@tennlee](https://github.com/tennlee)), Stephanie Chong ([@Steph-Chong](https://github.com/Steph-Chong)) and Robert J. Taggart ([@rob-taggart](https://github.com/rob-taggart)). 
```

Comment:
@durgals Can I please confirm (a) you are happy to be listed in the release notes and (b) you would like your name to appear as "Durga Shrestha"? If you would like any changes to be made, please let me know.